pyanaconda/iutil.py

```diff
@@ -38,6 +37,12 @@
 import random
 import functools
 from distutils import spawn
+import sys
+if os.name == 'posix' and sys.version_info[0] < 3:
+    import subprocess32 as subprocess
+else:
+    import subprocess
```

Comment:
Is this really required when you have subprocess32 as spec file dependency?

Reply:
I think we don't. Removed